Release 4.5.5 (#159)
blackbaud/skyux-modals: CHANGELOG.md

> # 4.5.5 (2021-07-12)

Reviewer: ```suggestion
# 4.5.5 (2021-07-13)
```

> - Fixed the modal component to properly skip disabled elements with a `tabIndex` attribute when focusing the first focusable child when the modal is created. [#158](https://github.com/blackbaud/skyux-modals/pull/158)

Reviewer: ```suggestion
- Fixed the modal component to skip disabled elements when applying focus after a modal is created. [#158](https://github.com/blackbaud/skyux-modals/pull/158)
```
I think the back-to-back-to-back prepositional phrases at the end made this hard to digest on the first read. I think this is easier to process f we delete the first prepositional phrase, tighten up the second, and change the preposition in the third to avoid repetition. ... That said, if these changes are too drastic, we can probably make less drastic changes to make this work: delete `properly`, change `a tabIndex attribute` to `tabIndex attributes` for parallel construction, add `on` after `focusing`, add `element` after `child`, and change the second `when` to `after`.

Author: My only concern is that the library was skipping disabled elements properly unless they had a `tabIndex` attribute. Thoughts?

Reviewer: I worked on this more after we talked and tried multiple variations. What do you think of this?
```suggestion
- Fixed the modal component so that it does not include disabled elements with `tabIndex` attributes among the focusable elements when a modal loads. [#158](https://github.com/blackbaud/skyux-modals/pull/158)
```

> - Fixed the modal component to properly skip disabled elements with a `tabIndex` attribute when focusing the first focusable child when the modal is created. [#158](https://github.com/blackbaud/skyux-modals/pull/158)

Reviewer 2: ```suggestion
- Fixed the modal component so that when the modal is created, it properly skips disabled elements with a `tabIndex` attribute when focusing the first focusable child. [#158](https://github.com/blackbaud/skyux-modals/pull/158)
```
